x-pack/plugin/esql/qa/testFixtures/src/main/resources/lookup-join.csv-spec

```diff
@@ -161,29 +165,36 @@ FROM employees
 | KEEP emp_no, language_code, language_name, country
 ;
```

Comment:
super nit: technically, the `MV_SORT` above is not required anymore. Arguably, it's better to remove it, as the `LOOKUP JOIN` should not re-order the multivalues that it looks up.

Applies similarly below.

Reply:
I'll have a look at these with a single node real cluster and see what I see. The ones without the MV_SORT should differ depending on how we run that and break. I'll get back to you here.

Reply:
Ok. The reason ES doesn't sort these as that the field is `text` and not `keyword`. `text` fields are loaded from the _source and we keep their ordering.

Reply:
Thanks for double checking, I learned something! :)